<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DrawGame</class>
 <widget class="QMainWindow" name="DrawGame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Крокодил</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QWidget" name="drawingArea" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="wordLabel">
        <property name="text">
         <string>Слово: -</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="statusLabel">
        <property name="text">
         <string>Статус: Ожидание начала игры</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="chatTextEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="messageLineEdit">
        <property name="placeholderText">
         <string>Введите сообщение...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="sendButton">
        <property name="text">
         <string>Отправить</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="startButton">
        <property name="text">
         <string>Начать игру</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="toolsLayout">
        <item>
         <widget class="QComboBox" name="colorComboBox">
          <item>
           <property name="text">
            <string>Черный</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Красный</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Зеленый</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Синий</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Желтый</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="widthSpinBox">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>20</number>
          </property>
          <property name="value">
           <number>3</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="clearButton">
          <property name="text">
           <string>Очистить</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="eraserButton">
          <property name="text">
           <string>Ластик</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
